core/src/dataio.cxx

```diff
@@ -18,7 +18,7 @@
 
 void
 g3_istream_from_path(boost::iostreams::filtering_istream &stream,
-    const std::string &path)
+                     const std::string &path, float timeout)
```

Comment:
Could you undo this whitespace change please?

Reply:
Fixed in https://github.com/CMB-S4/spt3g_software/pull/14/commits/23412a77a26b3ff80708447a8fc209b72057dc33.